lighty-core/lighty-controller/pom.xml

```diff
@@ -188,6 +188,11 @@
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
         </dependency>
+        <!--odl-aaa-cert-->
```

Comment:
Please move to be "together" with odl dependencies.

Reply:
hopefully fixed